### DISCUSSION TITLE
How to navigate to Dependency Dashboard on Azure DevOps

### DISCUSSION THREAD

**Discussion by crh225** [renovatebot/renovate#15679] [Request Help] [answered]
How do I navigate to Dependency Dashboard on Azure DevOps? I have the flag enabled.

**Comment by rarkins** [comment, marked as answer]
Dependency Dashboard is not supported for Azure DevOps

**Reply by crh225** [reply]
Can we update the docs to mention that?

**Reply by rarkins** [reply]
PRs welcome

**Reply by crh225** [reply]
https://github.com/renovatebot/renovate/blob/main/docs/usage/key-concepts/dashboard.md
https://docs.renovatebot.com/faq/#renovate-core-features-not-supported-on-all-platforms
https://github.com/renovatebot/renovate/issues/9592

I see it is documented already, my bad. Thanks!
